openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -35,12 +35,7 @@ <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
                     <div class="wrapper--content">
                         {% include "openassessmentblock/oa_submission_answer.html" with answer=submission.answer answer_text_label="The learner's response to the question above:" %}
 
-                        {% if submission.file_url %}
-                            <a href="{{ submission.file_url }}" class="submission--file">
-                                {% trans "The file associated with this response." %}
-                            </a>
-                            <span>{%  trans "Caution: This file was uploaded by another course learner and has not been verified, screened, approved, reviewed, or endorsed by edX. If you decide to access it, you do so at your own risk." %}</span>
-                        {% endif %}
+                        {% include "openassessmentblock/oa_uploaded_file.html" with file_upload_type=file_upload_type file_url=staff_file_url header="staff" class_prefix="staff-assessment" show_warning="true" %}
```

Comment:
Ack, this is using the new functionality in https://github.com/edx/edx-ora2/pull/833, so the string here will read "STAFF" unless both are merged together. I can fix this to use the old style if needed, but I'd rather just merge both PRs in at around the same time.